### PR TITLE
Allow all plugin properties to use groupName and grouping rendering options.

### PR DESCRIPTION
**Pull request by sjrd218**
Allow all plugin properties to use groupName and grouping rendering options.
Also fixes the name of a service in the Plugin api integration test.

### CODE REVIEW

File: core/src/main/java/com/dtolabs/rundeck/core/plugins/configuration/PluginAdapterUtility.java [hunk 1]
```diff
@@ -253,6 +253,11 @@ private static Property propertyFromField(final Field field, final PluginPropert
             for (RenderingOption renderingOption : field.getAnnotationsByType(RenderingOption.class)) {
                 pbuild.renderingOption(renderingOption.key(), renderingOption.value());
             }
+        } else {
+            for (RenderingOption renderingOption : field.getAnnotationsByType(RenderingOption.class)) {
+                if(renderingOption.key().equals(StringRenderingConstants.GROUP_NAME) || renderingOption.key().equals(StringRenderingConstants.GROUPING))
```
Comment (gschueler) [inline core/src/main/java/com/dtolabs/rundeck/core/plugins/configuration/PluginAdapterUtility.java]: any reason to restrict this to only "grouping/group_name" keys? i didn't realize non-string properties were skipping this annotation, probably we should just apply it to all properties

Comment (gschueler) [reply core/src/main/java/com/dtolabs/rundeck/core/plugins/configuration/PluginAdapterUtility.java]: originally, yes, however i think we should just add them all to the properties.  if they don't make sense for that property type they will be ignored, but better to allow it to be open ended for future options

**Comment by sjrd218** [comment]
Should this go in 3.2 or wait?

**Comment by gschueler** [comment]
ok for 3.2

**Pull request merged**